lib/pages/login-page.js

```diff
@@ -21,29 +22,32 @@ export default class LoginPage extends AsyncBaseContainer {
 		const userNameSelector = By.css( '#usernameOrEmail' );
 		const passwordSelector = By.css( '#password' );
 
-		await driverHelper.waitTillPresentAndDisplayed( driver, userNameSelector );
-		await this.driver.sleep( 1000 );
+		await this.driver.wait(
+			until.elementLocated( userNameSelector ),
+			this.explicitWaitMS,
+			'Could not locate the username or email field.'
+		);
+
 		await driverHelper.setWhenSettable( driver, userNameSelector, username );
-		await this.driver.sleep( 1000 );
 		await driver.findElement( userNameSelector ).sendKeys( Key.ENTER );
 
-		await this.driver.sleep( 1000 );
+		await this.driver.wait(
```

Comment:
Would `waitTillPresentAndDisplayed` work here instead? Reuse is always preferred

Reply:
Yes, as per my research in https://github.com/Automattic/wp-e2e-tests/pull/1698#discussion_r240554182, it is better to use `waitTillPresentAndDisplayed`. I will address it in the next PR. Thanks @bsessions85!